packages/features/tasker/tasks/sendEmail.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from "zod";
+
+const sendEmailPayloadSchema = z.object({
+  /**  */
+  to: z.string(),
+  /** The email template to send */
+  template: z.string(),
+  payload: z.string(),
+});
+
+export async function sendEmail(payload: string): Promise<void> {
+  try {
+    const parsedPayload = sendEmailPayloadSchema.parse(JSON.parse(payload));
+    console.log(parsedPayload);
+    const emails = await import("@calcom/emails");
+    const email = emails[parsedPayload.template as keyof typeof emails];
+    if (!email) throw new Error("Invalid email template");
+    // @ts-expect-error - TODO bring back email type safety
+    await email(parsedPayload.to);
+  } catch (error) {
+    // ... handle error
+    console.error(error);
+    throw error;
+  }
+}
```

Comment:
I noticed the payload schema only has a single `to` property but you're importing templates from the `email-manager`? Those methods send emails to organizers, team members, and attendees. I would say this isn't atomic. What if sending emails to the organizer works but sending to attendees fail? On retries the organizer would get multiple emails.

Reply:
That's way upper in the logic. task handler should not concern about these things.